vendor/bat-native-ads/src/bat/ads/ad_content_info.cc

```diff
@@ -85,51 +88,84 @@ bool AdContentInfo::FromValue(const base::Value::Dict& root) {
     placement_id = *value;
   }
 
-  if (const auto* value = root.FindString("creative_instance_id")) {
+  if (const auto* value = root.FindString("creativeInstanceId")) {
+    creative_instance_id = *value;
+  } else if (const auto* value = root.FindString("creative_instance_id")) {
+    // Migrate legacy
     creative_instance_id = *value;
   }
 
-  if (const auto* value = root.FindString("creative_set_id")) {
+  if (const auto* value = root.FindString("creativeSetId")) {
+    creative_set_id = *value;
+  } else if (const auto* value = root.FindString("creative_set_id")) {
+    // Migrate legacy
     creative_set_id = *value;
   }
 
-  if (const auto* value = root.FindString("campaign_id")) {
+  if (const auto* value = root.FindString("campaignId")) {
+    campaign_id = *value;
+  } else if (const auto* value = root.FindString("campaign_id")) {
+    // Migrate legacy
     campaign_id = *value;
   }
 
-  if (const auto* value = root.FindString("advertiser_id")) {
+  if (const auto* value = root.FindString("advertiserId")) {
+    advertiser_id = *value;
+  } else if (const auto* value = root.FindString("advertiser_id")) {
+    // Migrate legacy
     advertiser_id = *value;
   }
 
   if (const auto* value = root.FindString("brand")) {
     brand = *value;
   }
 
-  if (const auto* value = root.FindString("brand_info")) {
+  if (const auto* value = root.FindString("brandInfo")) {
+    brand_info = *value;
+  } else if (const auto* value = root.FindString("brand_info")) {
+    // Migrate legacy
     brand_info = *value;
   }
 
   if (const auto* value = root.FindString("brand_display_url")) {
```

Comment:
Would you mind keeping the correct value first, so we have mismatch only when migrating.

Reply:
Great spot! Fixed